### PR TITLE
Fix version of jdk for cd.yaml

### PR DESCRIPTION
**Pull request by timja**
@jenkinsci/github-admins

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]

File: workflow-templates/cd.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
       uses: actions/setup-java@v2
       with:
         distribution: 'adopt'
-        java-version: 1.8
+        java-version: 8
```
Comment (jglick) [inline workflow-templates/cd.yaml:53]: Does this work BTW, or do you need

```yaml
java-version: '8'
```

to force the value to be treated as a string?

Comment (timja) [reply workflow-templates/cd.yaml:53]: looks like this worked https://github.com/jenkinsci/jenkins-test-harness/commit/0f4bb0cb488469d8d2888d4c3ea8a6152e0592a2

Comment (jglick) [reply workflow-templates/cd.yaml:53]: Yes, it did. (There was a probably unrelated problem, what looks like a race condition after merging two PRs in quick succession: Release Drafter wound up including the second PR in the first release’s notes as well as the second’s.)

Comment (timja) [reply workflow-templates/cd.yaml:53]: I had that happen recently as well

**Pull request merged**